src/main/java/com/parasoft/findings/jenkins/coverage/api/metrics/source/SourceViewModel.java

```diff
@@ -54,7 +56,13 @@ public FileNode getNode() {
     @SuppressWarnings("unused") // Called by jelly view
     public String getSourceFileContent() {
         try {
-            return SOURCE_CODE_FACADE.read(getOwner().getRootDir(), id, getNode().getRelativePath());
+            String sourceFileContent = SOURCE_CODE_FACADE.read(getOwner().getRootDir(), id, getNode().getRelativePath());
+            // Localizing "tooltip" in source code.
+            sourceFileContent = sourceFileContent.replaceAll(PARASOFT_COVERAGE_ALL_BRANCHES_COVERED, Messages.All_Branches_Covered());
+            sourceFileContent = sourceFileContent.replaceAll(PARASOFT_COVERAGE_PARTIALLY_COVERED_AND_BRANCH_COVERAGE, Messages.Partially_Covered_And_Branch_Coverage());
+            sourceFileContent = sourceFileContent.replaceAll(PARASOFT_COVERAGE_COVERED_AT_LEAST_ONCE, Messages.Covered_At_Least_Once());
+            sourceFileContent = sourceFileContent.replaceAll(PARASOFT_COVERAGE_NOT_COVERED, Messages.Not_Covered());
```

Comment:
As the `sourceFileContent` here is normally a fragment of HTML, we may parse it as HTML, then modify the value of `data-html-tooltip` attribute to the localized tooltip, it can be more robust but may have performance impact.

Reply:
Since `String` is immutable in Java, the implementation here may have performance issues.